test/e2e/azure_logcollector.go

```diff
@@ -356,6 +359,16 @@ func windowsNetworkLogs(execToPathFn func(outputFileName string, command string,
 	}
 }
 
+func windowsCrashDumpLogs(execToPathFn func(outputFileName string, command string, args ...string) func() error) []func() error {
+	return []func() error{
+		execToPathFn(
+			// capture any crashdump files created by windows into a .tar to be collected via sftp
+			"tar-crashdumps.txt",
+			"$p = 'c:\\localdumps' ; if (Test-Path $p) { tar.exe -cvzf c:\\crashdump.tar $p 2>&1 } else { Write-Host \"No crash dumps found at $p\" }",
```

Comment:
I was expecting the file in https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_cluster-api-provider-azure/2510/pull-cluster-api-provider-azure-windows-containerd-upstream-with-ci-artifacts/1552455520860770304/artifacts/clusters/capz-conf-olydfh/machines/capz-conf-olydfh-md-win-99c6c8cbc-x564z/tar-crashdumps.txt  to have "No crash dumps found at" but not seeing it.  any thoughts?

Reply:
The path does exist (we make it in the kubeadm action) so I would expect the file to show some output from tar.exe.
Will check on this.

Reply:
I believe i have addressed this.